identity_platform/resource_config.go

```diff
@@ -188,6 +220,18 @@ func hydrate(diags diag.Diagnostics, config *idp.Config, d *schema.ResourceData)
 		diag.FromErr(err)
 	}
 
+	arr = TerraformListType{
+		{
+			"sendEmail": TerraformObjectType{
```

Comment:
```suggestion
			"sendEmail": Object{
```
Is there any reason to create an alias on an alias?

Reply:
lol I potated.